### ISSUE TITLE
Cant shoot in vehicle when stationary

### ISSUE THREAD

**Issue by Ramd1nan1l** [ThirteenAG/GTAIV.EFLC.FusionFix#667]
### Game version this happens in

v1.2.0.59

### What area of the game

The bug isn't limited to any section of the game.

### What should happen

I cant shoot my weapon when the car I'm in isn't moving. As soon as the car comes to a stop Niko stops shooting. This is only on gamepad, with the controller config set to standard, when tested on the classic controller config everything works fine.

### Checklist

- [X] Test in the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already fixed.
- [X] [Search for other reports](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same issue.
- [X] Include logs or screenshots of issue.

**Comment by sTc2201** [comment]
Does this have anything to do with FusionFix though? Nowhere do you mention that it doesn't happen without it, issues are only for FusionFix related issues.

**Comment by Ramd1nan1l** [comment]
Yes, this only happens with fusion fix, I didn't think I needed to clarify that being where I posted this.  And no, it occurs for ALL vehicles, I'm aware this exists in the vanilla game for police vehicles. 
One thing I noted is that when starting a new game, this issue does NOT occur up until you pass the first mission, and afterwards it starts to persists, again ,on ALL vehicles. 
Just tested the vanilla game again, and this bug isn't present, however when installing FusionFix again,  the issue was present again.
